.github/workflows/lint.yml

```diff
@@ -152,6 +146,10 @@ jobs:
         if: always() && steps.install_shellcheck.outcome == 'success'
         run: |
           tools/run_shellcheck.sh .extracted_scripts .jenkins/pytorch
+      - name: Check that jobs will be cancelled
+        if: always() && steps.extract_scripts.outcome == 'success'
```

Comment:
out of curiosity, why does this depend on the `extract_scripts` step?

Reply:
ah I mixed that up with the step to generate the workflows, it's fixed now